app/javascript/controllers/locationsearch_controller.js

```diff
@@ -0,0 +1,37 @@
+import { Controller } from '@hotwired/stimulus'
+import TomSelect from 'tom-select'
+
+export default class extends Controller {
+  static values = {
+    options: Object
+  };
+
+  connect() {
+    this.accessToken = this.element.dataset.apiKey;
+    this.tomselect = new TomSelect('#locationInput', {
+      valueField: 'text',
+      labelField: 'text',
+      searchField: 'text',
+      load: (search, callback) => {
+        this.searchMapbox(search, callback)
+      }}
+    );
+  }
+
+  async searchMapbox(search, callback) {
+    if(search == "")
+      callback([]);
+
+    const url =`https://api.mapbox.com/geocoding/v5/mapbox.places/${search}.json?access_token=${this.accessToken}`;
```

Comment:
Vi får opp masse steder her som er helt irrelevante. Er det mulig å filtrere dem på "Norge"?

![image](https://user-images.githubusercontent.com/14905290/138305119-0b482318-bb77-4cb1-b595-2aae0ed5ae79.png)

Reply:
Etter litt ekspeimenter, tror jeg det beste er å bruke en annen datakilde - f.eks. https://ws.geonorge.no/stedsnavn/v1/#/default/get_sted 

Der får vi alle stedsnavn i Norge, med norsk standard for hva som er det geografiske området og "sentrum" for stedet.